modules/ROOT/pages/upgrade-policies-proxies.adoc

```diff
@@ -0,0 +1,128 @@
+= Upgrade Java for Policies and API Proxies 
+
+Upgrade your policies and API proxies to be compatible with Java 17.
+
+[[upgrading-automated-policies]]
+== Upgrading Automated Policies
+
+Upgrade your automated policies (included and custom) to Java 17.
+
+[[upgrading-included-automated-policies]]
+=== Upgrading Included Automated Policies
+
+To upgrade automated xref:mule-gateway::policies-included-directory.adoc[MuleSoft included policies] to Java 17:
+
+. In API Manager, go to *Automated Policies*.
+. In *Policy version*, select the latest version of the included policy.
+. In *Rule of application*:
+  * To apply included policies to all Mule runtimes, select *All runtimes* and click *Apply*.
+  * To apply the included policies to Flex Gateways only, select *Flex Gateways only* and click *Apply*.
+  * To apply the included policies to Mule Gateways only:
+  ** Select *Mule Gateways only*.
+  ** In *From version* and *To*, enter the Mule version range.
+  ** In *Java versions*, select *All Java versions* or *Choose specific Java versions*. 
+  ** Click *Apply*.
+
+[[upgrading-custom-automated-policies]]
+=== Upgrading Custom Automated Policies
+
+To upgrade custom automated policies to Java 17:
+
+. In API Manager, go to *Automated Policies*.
+. Upgrade the required extensions and connectors to Java 17.
+ * If your custom policy has Anypoint Connectors, see xref:java-support.adoc#anypoint-connectors-and-modules[Upgrading Anypoint Connectors and Modules].
+ * If your custom policy has custom connectors, see <<Upgrading Custom Connectors INSERT LINK>>.
+. xref:gateway::policies-custom-java-version.adoc[Upgrade the custom policy (link to new page in Gateway)].
+. xref:gateway::policies-custom-package.adoc[Package the custom policy].
+. xref:gateway::policies-custom-upload-to-exchange.adoc[Republish the custom policy to Exchange].
+. Apply the new version of the custom policy that is compatible with Java 17 to the API instance:
+ .. Go to *Policies*.
+ .. In *Applied policy status*, verify that the latest definition is applied.
+ * If the latest definition is applied, click on the More (...) menu and select *Check for implementation updates*.
+ * If the latest definition is not applied, update to the latest definition.
+
+[[upgrading-api-level-policies]]
+== Upgrading API-Level Policies
+
+Upgrade your API-level policies (included and custom) to Java 17.
+
+[[upgrading-included-api-level-policies]]
+=== Upgrading Included API-Level Policies
+
+To upgrade API-level xref:mule-gateway::policies-included-directory.adoc[MuleSoft included policies] to Java 17:
+
+. In API Manager, go to *API Administration > <API Name> > Policies > Add a policy* to xref:gateway::policies-included-apply.adoc[apply policies] at the individual API level. 
```

Comment:
Wondering if there's any alternative to **<API Name>** because the angle brackets might get mixed with the > indicating next. I think asciidoc renders an arrow if you use --> 
Otherwise, maybe use curly brackets here? to: *API Administration > {API Name} > Policies > Add a policy*
I'll leave you to decide what's best in this situation. Making no change is also a valid option.